faq/prooffacehelp.php

```diff
@@ -596,12 +637,13 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 <P><A NAME="ikeys"> </A> <P><FONT SIZE="+1">Accelerator Keys (accesskeys)</FONT><P></TD></TR>
 <TR>
 <TD VALIGN="TOP" COLSPAN="2">
-Several of the commonly used interface buttons have been assigned an accesskey value.
+<p>These accelerator keys may not work for everyone.</p>
```

Comment:
I would say: ... everyone, and the modifier key (shown as Alt below) may be different depending on your Operating System and browser.

Reply:
Please coordinate with @lhamilton1 on the wording of this, whatever you both decide is fine with me and needs to be updated on the wiki as well.

Reply:
Updated.